TicTacToe.py

```diff
@@ -228,38 +230,39 @@ def advancedGameSettings(self, settingToChange):
 					player0 = input('What do you want first move icon to be? (Traditionaly X): ')
 					if len(player0) != 1:
 						print("Please enter a single character for the player icon")
-				
+
 				player1 = 'too long'
 				while len(player1) != 1:
 					player1 = input('What do you want second move icon to be? (Traditionaly O): ')
 					if len(player0) != 1:
 						print("Please enter a single character for the player icon")
 
 				self.updatePlayerIcons(player0, player1)
-				
 
 	def gameSettingsPrompt(self):
 		"""Prints messages to allow the user to select number of players then choice which icon
 		Takes no inputs
 
 		:returns: a tuple of the functions to call to process the 0 and 1 player moves
 		"""
-		
+
 		# Prompts for how many human players there will be
 		num_players = 0
 		while num_players != 1 and num_players != 2:
 			num_players = input("Enter the number of players (1 or 2): ")
 			if num_players.isnumeric():
 				num_players = int(num_players)
 			elif num_players == 'change icons':
-					self.advancedGameSettings(num_players)
-		
+				self.advancedGameSettings(num_players)
+
 		# if user selected single player
 		if num_players == 1:
 			# choose the player icon
 			player_choice = self.BLANK_POS_ICON
 			while player_choice != self.PLAYER_0_ICON and player_choice != self.PLAYER_1_ICON:
-				player_choice = input(f"{self.PLAYER_0_ICON}s plays first, do you want to be {self.PLAYER_0_ICON} or {self.PLAYER_1_ICON}? ")
+				player_choice = input(
+					f"{self.PLAYER_0_ICON}s plays first, do you want to be{self.PLAYER_0_ICON} or {self.PLAYER_1_ICON}? "
```

Comment:
This is exactly what I had in mind for fixing this one 👍

Reply:
Awesome!